tests/mysql/test_mysql.py

```diff
@@ -455,6 +454,19 @@ def mysql_connector_with_ssl():
     )
 
 
+@pytest.fixture
+def mysql_connector_with_ssl_bundle():
+    return MySQLConnector(
+        name='mycon',
+        host='localhost',
+        port=3306,
+        user='ubuntu',
+        password='ilovetoucan',
+        ssl_ca='-----BEGIN CERTIFICATE-----\nsomething\n-----END CERTIFICATE-----\n-----BEGIN CERTIFICATE-----\nsomething\n-----END CERTIFICATE-----',
```

Comment:
```suggestion
        ssl_ca='-----BEGIN CERTIFICATE----- something -----END CERTIFICATE----- -----BEGIN CERTIFICATE----- something -----END CERTIFICATE-----',
```

To simulate what we could be receveiving